client/components/tinymce/plugins/contact-form/style.scss

```diff
@@ -78,3 +78,20 @@
 		color: $orange-fire;
 	}
 }
+
+.editor-contact-form-modal-field__edit-wrapper {
+	height: 100%;
+
+	button {
+		width: 100%;
+	}
+}
+.editor-contact-form-modal-field__edit-wrapper.foldable-card__expand .editor-contact-form-modal-field__edit {
```

Comment:
We try to avoid this much specificity, but it might not be avoidable in this case.